src/frontend/packages/lib_components/src/hooks/stores/useJwt/index.ts

```diff
@@ -52,34 +56,45 @@ export const persistentStore = create<JwtStoreInterface>((set, get) => ({
   getJwt() {
     const jwt = localStorage.getItem(JWT_KEY) || undefined;
     if (jwt !== get().jwt) {
-      set((state) => ({ ...state, jwt, internalDecodedJwt: undefined }));
+      get().setJwt(jwt);
```

Comment:
This is wrong IMO because the `internalDecodedJwt` is never reset. If after I call the `getDecodedJwt` then the value will be wrong.

Reply:
Yes it is, `getJwt` call `setJwt` that call `setDecodeJwt`.